dependencyManagement/build.gradle.kts

```diff
@@ -7,15 +7,13 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
-val otelInstrumentationVersion = "2.10.0-alpha"
-
 val DEPENDENCY_BOMS = listOf(
   "com.fasterxml.jackson:jackson-bom:2.18.2",
   "com.google.guava:guava-bom:33.3.1-jre",
   "com.linecorp.armeria:armeria-bom:1.31.2",
   "org.junit:junit-bom:5.11.3",
   "io.grpc:grpc-bom:1.68.2",
-  "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:${otelInstrumentationVersion}",
+  "io.opentelemetry:opentelemetry-bom-alpha:1.44.1-alpha",
```

Comment:
I didn't update to 1.45.0 in order to ensure everything works smoothly when renovate updates it after this PR is merged